### DISCUSSION TITLE
Existing VMs unable to run and new installation of Ubuntu failed after macOS is patched to the latest version 4.5.3.

### DISCUSSION THREAD

**Discussion by tankwong** [utmapp/UTM#6615] [Linux]
**Describe the issue**  
After upgrading MacOS to the latest patch version 4.5.3, all my existing VMs running using ubuntu 22.04 for ARM (https://mac.getutm.app/gallery/ubuntu-20-04 ) behaved in a non-consistent manner e.g. sometimes shared folders could not be accessed, sometimes external internet access could not be accessed, keystrokes at command prompt would hanged etc.

Then I have decided to install a brand new ubunto instance and this time, the following errors occurs right on the onset of loading the server .iso.  

Hope to get this resolved.  Thanks.

**Configuration**  
* UTM Version: Version 4.5.3 (99)
* macOS Version: Sonoma 14.6.1 (23G93)
* Mac Chip (Intel, M1, ...):  M2 Pro

** Crash log when trying to install ubuntu server 24.04 iso, similar to 22.04.4 and 20.04.5 as well. **  
QEMU error: QEMU exited from an error: qemu-aarch64-softmmu: could not load kernel '/Users/williamtan/Library/Containers/com.utmapp.UTM/Data/Documents/ubuntu 24.04 (base 20240817).utm/Data/ubuntu-24.04-live-server-arm64.iso'

**Comment by MaksHant** [comment]
Добрый день, идет 7 июня 2025 года и такая же ошибка вылезает у меня при загрузке Kali Linux. Есть ли решение данной ошибки?
